src/Link.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { Text } from 'react-native';
+import PropTypes from 'prop-types';
+
+import normalize from './helpers/normalize';
+import GalioTheme, { withGalio } from './theme';
+
+function Typography({
+    style,
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6,
+    p,
+    body,
+    small,
+    muted,
+    neutral,
+    size,
+    color,
+    bold,
+    italic,
+    center,
+    children,
+    styles,
+    theme,
+    ...rest
+  }) {
+    return (
+        <TouchableOpacity onPress={() => Linking.openURL(url)}>
+            <Text style={[
```

Comment:
This components should be our `<Text />` component, not the basic one from the core 'react-native' library. By using our own `<Text />` component, this one will inherit all of those props, so this could save a lot of space in the function.